docs/maui/converters/enumtointconverter.md

```diff
@@ -0,0 +1,104 @@
+---
+
+# EnumToIntConverter
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `EnumToIntConverter` is a converter that allows you to convert a standard `Enum` (extending int) to its underlying primitive `int` type. It is useful when binding a collection of values representing an enumeration type with default numbering to a control such as a `Picker`.
+
+For localization purposes or due to other requirements, the enum values often need to be converted to a human-readable string. In this case, when the user selects a value, the resulting `SelectedIndex` can easily be converted to the underlying `enum` value without requiring additional work in the associated ViewModel.
+
+## Syntax
+
+### XAML
+
+The `EnumToIntConverter` can be used as follows in XAML:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="MyLittleApp.MainPage">
+    <ContentPage.Resources>
+         <ResourceDictionary>
+             <toolkit:EnumToIntConverter x:Key="EnumToIntConverter" />
+         </ResourceDictionary>
+    </ContentPage.Resources>
+
+    <VerticalStackLayout Padding="10,10" Spacing="10">
+            <Label Text="The EnumToIntConverter is a converter that allows users to convert a standard enum (extending int) to its underlying primitive int type." 
+                   TextColor="{StaticResource NormalLabelTextColor}" />
+            <Label Text="Selecting a value from the picker will change the enum property in the view model" 
+                   TextColor="{StaticResource NormalLabelTextColor}" />
+            <Picker ItemsSource="{Binding AllStates}" 
+                    SelectedIndex="{Binding SelectedState, Converter={StaticResource EnumToIntConverter}}" 
+                    TextColor="{StaticResource NormalLabelTextColor}" />
+            <Label Text="This label binds to the SelectedIndex property of the picker, both use EnumToIntConverter, so no int properties are necessary in ViewModel"
+                   TextColor="{StaticResource NormalLabelTextColor}" />
+            <Label Text="{Binding Path=SelectedState, Converter={StaticResource EnumToIntConverter}}" 
+                   TextColor="{StaticResource NormalLabelTextColor}" />
+        </VerticalStackLayout>
+</ContentPage>
+```
+
+
+
+### C#
+
+The `EnumToIntConverter` can be used as follows in C#:
+
+```csharp
+class EnumToIntConverterPage : ContentPage
+{
+    public EnumToIntConverterPage()
+    {
+        Content = new StackLayout
+    {
+      Children = 
+      {
+        new Picker()
+          .Bind(Picker.ItemSource, nameof(ViewModel.AllStates)
+          .Bind(Picker.SelectedIndex, nameof(ViewModel.SelectedState),
+
+          new Label()
+            .Bind(Label.TextPropoerty, nameof(ViewModel.SelectedState), converter: new EnumToIntConverter()),
```

Comment:
This looks remarkably similar to the `C# Markup` section below 😄. 

I am happy to try and convert to the standard way of doing things if it helps

Reply:
I cut and paste that section from the Proposal as I don't tend to build my UI in C#, I must admit I am a little confused as to the difference between the C# and Markup so happy to be shown what the difference is.

Reply:
Actually I see what you mean by the difference and think I understand what your looking for. Let me have another crack at it in the morning.

Reply:
OK thanks, let me know how you get on. Basically the thought behind the C# section and then the C# Markup section is to show how great the .Markup package is for developers that like to build their UI in C# only. Actually I suspect this example will show just how much simpler the C# Markup package makes it.